idea_inspection.bat

```diff
@@ -0,0 +1,37 @@
+@echo off
+
+::----------------------------------------------------------------------
+:: IntelliJ IDEA inspections for checkstyle.
+::----------------------------------------------------------------------
+SET PROJECT_DIR=%CD%\
+SET INSPECTIONS_PATH=%CD%\config\intellij-idea-inspections.xml
+SET RESULTS_DIR=%CD%\inspection-results
+SET NOISE_LVL=v1
+SET IDEA_LOCATION=
+SET IDEA_PROPERTIES=%CD%\config\idea.properties
+
+::Check IDEA_HOME env variable
+IF EXIST %IDEA_HOME% SET (
+    SET IDEA_LOCATION=%IDEA_HOME%
+    goto run
+) ELSE (
+    echo IDEA_HOME variable not found.
+)
+
+::Try to search in path
+FOR /f "delims=" %%i IN ('"where idea.bat"') DO SET IDEA_LOCATION="%%i"
+if [%IDEA_LOCATION%] NEQ [] (
+    goto run 
+) ELSE (
+    echo IntelliJ IDEA was not found in path
+)
+
+::Ask user for input
```

Comment:
No interactions please. Fail with error if smth is not defined

Reply:
done